pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -1,11 +1,23 @@
 # -*- coding: utf-8 -*-
 
+from itertools import chain
+from pkg_resources import resource_string
+import re
+
 import numpy as np
 
 from pyfr.integrators.dual.pseudo.base import BaseDualPseudoIntegrator
 from pyfr.util import proxylist
 
 
+def _get_coefficients(scheme):
```

Comment:
_get_coefficients_from_txt may be a better name?  get_coefficients seems a tad generic.